src/components/app/stylesheets/base.scss

```diff
@@ -11,7 +11,7 @@
 }
 
 html {
-  font-size: math.percentage(10 / 16);
+  font-size: math.percentage(math.div(10, 16));
```

Comment:
💡 Stylelint packages were giving off these warnings; using sass packages is the way forward.